### DISCUSSION TITLE
klt-principle-can-save-your-business

### DISCUSSION THREAD

**Discussion by giscus** [bartoszgorka/bartoszgorka.github.io#117] [Announcements]
# klt-principle-can-save-your-business

Selling is not an easy job. The customers don’t care about you. What matters is how you can solve their problems. People often recommend and buy from people they know well, like, and trust. See how to use the KLT principle in your business.

https://bartoszgorka.com/klt-principle-can-save-your-business

**Comment by masoudgs** [comment]
The article was really helpful and Thanks for sharing your discovery.
